addons/mail/data/mail_activity_data.xml

```diff
@@ -12,6 +12,12 @@
             <field name="delay_count">2</field>
             <field name="sequence">6</field>
         </record>
+        <record id="mail_activity_data_scheduled_call" model="mail.activity.type">
+            <field name="name">Scheduled Call</field>
+            <field name="icon">fa-phone</field>
+            <field name="delay_count">2</field>
+            <field name="sequence">7</field>
+        </record>
```

Comment:
How is that different from `mail_activity_data_call`?
If we really need an extra one, don't forget to integrate it in `voip` module (`<field name="category">phonecall</field>` should be enough)